tox.ini

```diff
@@ -2,7 +2,7 @@
 envlist=
     py{26,27,33,34,35,36,37,38}-test,
     py{26,27,33,34,35,36,37,38}-requests,
-    py{26,27}-flask,
+    py{26,27,33,34,35,36,37,38}-flask,
```

Comment:
Mention of 33 here but not 3.3 in the yml file.  Is that significant?

Reply:
Travis didn't like running tox with Python 3.3 (e.g. https://travis-ci.com/github/bugsnag/bugsnag-python/jobs/338799201)

I figured it was nicer to define them here to show what could be run on 3.3 and they pass for me locally on 3.3 via tox

I think I gave up too easily getting it to run on Travis though, I'll spend a little bit more time trying to make it work more like the other versions

Reply:
Yeah, I definitely gave up too easily — it's now running with Tox on Travis :) Still a bit of a special case, but better than it was!